index.html

```diff
@@ -168,11 +168,16 @@ <h1>Hacker Escape Rooms</h1>
                     vos
                     omnino desit illud quo solo felicitatis libertatisque perficiuntur.</p>
             </div>
+            
             <div class="main__storyBox--button button darkBlue">
-                <a href="about-us.html">Read the full story</a> <!--Check font size and styling -->
+                <a href="about-us.html">
```

Comment:
Swap line 173 with 172 to catch the whole div and thus adding the animation to the button and not only the text